x/cron/keeper/keeper.go

```diff
@@ -27,6 +27,8 @@ var (
 
 	MetricLabelSuccess      = "success"
 	MetricLabelScheduleName = "schedule_name"
+
+	schedulesExecutionStages map[string]map[types.ExecutionStage]struct{}
```

Comment:
please refactor to Keeper.Store usage based on our discussion

Reply:
We've discussed this with Mike and decided to forego the option of executing schedule in several stages.